src/Plugin/Field/FieldFormatter/PlayerFormatter.php

```diff
@@ -228,23 +230,25 @@ public static function defaultSettings() {
   protected function fetchPlayerOptions(): array {
     $options = [];
     $bundle = $this->fieldDefinition->getTargetBundle();
-    /** @var \Drupal\media\Entity\MediaType $type */
-    $type = $this->entityTypeManager->getStorage('media_type')->load($bundle);
-    /** @var \Drupal\media_mpx\Plugin\media\Source\Media $source_plugin */
-    $source_plugin = $type->getSource();
-
-    $factory = $this->dataObjectFactoryCreator->forObjectType($source_plugin->getAccount()->getUserEntity(), 'Player Data Service', 'Player', '1.6', $source_plugin->getAccount());
-    $query = new ObjectListQuery();
-    $sort = new Sort();
-    $sort->addSort('title');
-    $query->setSort($sort);
-
-    /** @var \Lullabot\Mpx\DataService\Player\Player[] $results */
-    $results = $factory->select($query);
-
-    foreach ($results as $player) {
-      if (!$player->getDisabled()) {
-        $options[(string) $player->getId()] = $player->getTitle();
+    if ($bundle) {
+      /** @var \Drupal\media\Entity\MediaType $type */
+      $type = $this->entityTypeManager->getStorage('media_type')->load($bundle);
+      /** @var \Drupal\media_mpx\Plugin\media\Source\Media $source_plugin */
+      $source_plugin = $type->getSource();
+
+      $factory = $this->dataObjectFactoryCreator->forObjectType($source_plugin->getAccount()->getUserEntity(), 'Player Data Service', 'Player', '1.6', $source_plugin->getAccount());
+      $query = new ObjectListQuery();
+      $sort = new Sort();
+      $sort->addSort('title');
+      $query->setSort($sort);
+
+      /** @var \Lullabot\Mpx\DataService\Player\Player[] $results */
+      $results = $factory->select($query);
+
+      foreach ($results as $player) {
```

Comment:
If each result is a player option, shouldn't it be called `$player_option`, or `$option`?